modules/apps/account/account-admin-web/src/main/resources/content/Language.properties

```diff
@@ -52,6 +52,7 @@ there-are-no-users-associated-with-this-account=There are no users associated wi
 there-are-no-users-associated-with-this-role=There are no users associated with this role.
 this-account-does-not-have-a-parent-account=This account does not have a parent account.
 this-account-does-not-have-a-valid-domain=This account does not have a valid domain.
+this-email-has-invalid-domain=This email has invalid domain.
```

Comment:
<img width="613" alt="Screen Shot 2020-06-10 at 3 58 44 PM" src="https://user-images.githubusercontent.com/5168351/84327114-71d9c380-ab33-11ea-87d1-71071a4ff637.png">

The warning message in the mockup is `You are adding an email with invalid domain`, but this doesn't sound right if you're editing the user.  And `This email domain is not allowed` sounds like the the email domain cannot be used.

@dejuknow do you have any other suggestion for the warning message?

Reply:
how about: "This email address 

Can you make the text dynamic? If so, how about: "2binc.com is not a valid domain for this account."

If not, how about: "This email address domain is not a valid domain for this account."

Reply:
Where does the text "This email domain is not allowed" show up?

Reply:
This only works when adding new account users because adding is for one specific account.

When editing users, we're validating the email address across all accounts that the user belongs to.

Reply:
ahh ... hm. How about: "The email address domain is not valid for all associated accounts."

or if we can make it dynamic, "The email address domain is not valid for the following associated accounts: Volkswagon USA, Volkswagon Intl."

Reply:
Dynamic: "2b.com is not a valid domain for the following accounts: Volkswagon USA, Volkswagon Intl".

Static: "The email address domain is not valid for all associated accounts."

Reply:
We can always pass in a different message to the javascript component if it's going to be used in multiple places.

We can use one for editing a user and one for adding a new account user.

Reply:
Ok in the process of adding a user to an account, when the frontend email domain validation fails, I think it would be really really helpful to include the valid domains in the warning message. I saw the current message and immediately thought "which ones did I set again?".